src/stores/widgets/SpectralProfileWidget/SpectralProfileWidgetStore.ts

```diff
@@ -316,16 +316,12 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
         this.intensityConversion = undefined;
         this.setIntensityUnit(this.effectiveFrame?.unit);
 
-        reaction(
-            () => this.effectiveFrame,
-            frame => {
-                if (frame) {
-                    this.momentRegionId = RegionId.ACTIVE;
-                    this.setIntensityUnit(frame.unit);
```

Comment:
is there any effect after removing this line?

Reply:
I found that the intensity unit options in conversion and the unit of y axis in spectral profiler won't be updated when switching images. Should add the removing line back.

Reply:
revised